### PR TITLE
Report for May 11 upgrade

### PR DESCRIPTION
**Pull request by shayanb**
Report for the https://github.com/PufferFinance/Deployments-and-ACL/pull/3 address updates.

### CODE REVIEW

File: docs/reports/2024-05-10-optimization-bug-fix-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+# Timeline 
+## Friday, May 11, 2024
+
+- 1:02 am - Initial issue was raised as the `totalAssets` was returning 0 on Puffer Vault
+    - Impact: totalAssets value is required to calculate the pufETH/ETH conversion ratio ("price"). This issue has resulted in some DeFi integrations being implicitly paused, as the view calls would revert internally
+- 1:23 am - War room began to check if pausing the contracts is needed
+    - BlockSec and Creed auditors were contacted
+- 1:45 am - Verified that all the funds are safe and the latest provisioned validators are using the correct protocol controlled withdrawal addresses.
+- 2:25 am - The issue was found to be the underflow in the `PufferOracle.getLockedEthAmount()`, introduced in a storage optimization code change, which now is resulting in the totalAsset call failure
```
Comment (JasonVranek) [inline docs/reports/2024-05-10-optimization-bug-fix-upgrade.md:26]:
```suggestion
- 2:25 am - The issue was found to be the overflow in the `PufferOracle.getLockedEthAmount()`, introduced in a storage optimization code change, which now is resulting in the totalAsset call failure
```

File: docs/reports/2024-05-10-optimization-bug-fix-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+- Stop provisioning new validator - (done at 2:35 AM)
+    - Removed the event trigger to stop the cronjob — stopping guardians to provision validator
+
+## Deployments
+
+- new **PufferOracleV2** needs to be deployed, with the fix uint256 for active validators
+    - Import the current Oracle variables as constructor variables for the new contract
+        - _epochNumber = 268828
+        - _totalNumberOfValidators = 927122
+        - _numberOfActivePufferValidators = 174 //Double check before deployment
```
Comment (JasonVranek) [inline docs/reports/2024-05-10-optimization-bug-fix-upgrade.md:150]:
```suggestion
        - _numberOfActivePufferValidators = 174
```

File: docs/reports/2024-05-10-optimization-bug-fix-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        - _setMintPrice([3098470956104156](https://etherscan.io/unitconverter?wei=3098470956104156)) //latest
+
+The following contracts need to be updated since the `PufferOracleV2` address is an immutable variable set in the constructor:
+
+- new **PufferVaultV2** implementation
+    - Only swap the PufferOracleV2 address in the constructor
+- new **ValidatorTicket** implementation
+    - Only swap the PufferOracleV2 address in the constructor
+- new **OperationCoordinator**
```
Comment (JasonVranek) [inline docs/reports/2024-05-10-optimization-bug-fix-upgrade.md:159]:
```suggestion
- new **OperationsCoordinator**
```

File: docs/reports/2024-05-10-optimization-bug-fix-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+- https://etherscan.io/tx/0x5a2f4e83e19e6b33fce07710d5acae5199156d0c82168386cb5e25cc29a0f575
+- https://etherscan.io/tx/0x370fde4254e711576873adb054bdce8c217cfe7822acef8d353451a0a46f0698
+- https://etherscan.io/tx/0x9d4db5af8cb489b899bdf7d699a281a013d730afc2f371fb708dc9ad771bc203
+- https://etherscan.io/tx/0x2c485d8b964ca8bb86de04d520befcab5d5ccc24ed99766c234a57decbe2b7b5
+
+## Assign roles
+
+https://github.com/PufferFinance/Deployments-and-ACL/blob/main/docs/access-control/contracts_and_functions.md
+
+- _setupCoordinatorAccess()
```
Comment (JasonVranek) [inline docs/reports/2024-05-10-optimization-bug-fix-upgrade.md:183]: think  there were more roles

**Review by JasonVranek — APPROVED** [review]

File: docs/reports/2024-05-10-optimization-bug-fix-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    - Removed the event trigger to stop the cronjob — stopping guardians to provision validator
+
+## Deployments
+
+- new **PufferOracleV2** needs to be deployed, with the fix uint256 for active validators
+    - Import the current Oracle variables as constructor variables for the new contract
+        - _epochNumber = 268828
+        - _totalNumberOfValidators = 927122
+        - _numberOfActivePufferValidators = 174 
+        - _setMintPrice([3098470956104156](https://etherscan.io/unitconverter?wei=3098470956104156)) //latest
```
Comment (bxmmm1) [inline docs/reports/2024-05-10-optimization-bug-fix-upgrade.md:151]:
```suggestion
        - _setMintPrice([3098470956104156](https://etherscan.io/unitconverter?wei=3098470956104156)) (latest value)
```

**Review by bxmmm1 — APPROVED** [review]

**Pull request merged**